ros/angel_utils/scripts/bag_extractor.py

```diff
@@ -570,6 +574,9 @@ def handle_image_msg(self, msg: Image) -> None:
             else:
                 rgb_image = bridge.imgmsg_to_cv2(msg, msg.encoding)
 
+            if "rgb" in msg.encoding:
+                rgb_image = cv2.cvtColor(rgb_image, cv2.COLOR_BGRA2RGBA)
```

Comment:
Is L575 not outputting an RGB image already when the `msg.encoding` is "rgb"? It looks like  you're converting form "BGR" to "RGB" here, but I would have expected `rgb_image` to already be in RGB at this point (its just maybe missing the alpha channel?).